diff --git a/scripts/setup.sh b/scripts/setup.sh
index 3594690..c23bfda 100755
--- a/scripts/setup.sh
+++ b/scripts/setup.sh
@@ -997,7 +997,6 @@ EOF
            touch "$serials_file" &&
            chown $wwwuser "$serials_file" &&
            chown $wwwuser "$htpasswd_file" &&
-	   create_sudo_configuration &&
            if [ "$enable_mkeventd" = yes ]
            then
 	       if [ -z "$YES" ] ; then echo -n "(Compiling Event Console binaries..." ; fi
